### DISCUSSION TITLE
No Properties + Flicking screen

### DISCUSSION THREAD

**Discussion by kimjboland** [valinet/ExplorerPatcher#1963] [General]
Just tried a reinstall of the app. (latest version)
right click on taskbar only causes screen to flicker and reset. thus not able to change any settings. Is there another way to access properties?

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.

**Comment by DdAArt** [comment]
I tried this on 2 different win11 installs; one reinstall, one new. When properties is enabled there is flicker and no taskbar menu on right mouseclick (as described). If taskbar context menu is disabled, in advanced, then standard win11 taskbar shows up as normal.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

**Comment by DdAArt** [comment]
These are my current settings. Properties are disabled. Enabling properties and right clicking with win11 taskbar results in 3 resets and explorer patcher crash, as stated elsewhere. 

I've tried  cleaning registry and reinstalling 4 or 5 times. It takes a minute or so but current symbols always download fine. Taskbar defaults to win 10 style manually switching it to win11 style causes the problems.

<img width="368" alt="Screenshot 2023-09-01 115323_vert" src="https://github.com/valinet/ExplorerPatcher/assets/35466545/9419eb39-0281-4464-ba29-d7ece40efb4d">

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1828

Please close this one.

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Reply by DdAArt** [reply]
Doing as you suggest is what causes it to reset and crash. Only leaving it as I have in the screenshot allows rightclick of taskbar while in windows 11 mode.  My choices are either windows 10 mode and "properties" activated or windows 11 mode and "properties" deactivated.

but you are correct in that this is a duplicate of problem many others have.

**Reply by pyrates999** [reply]
Please close this then.
